scripts/merge-and-build.sh

```diff
@@ -143,28 +143,28 @@ cd ${WORKPATH}
 rm -rf ose
 git clone git@github.com:openshift/ose.git
 cd ose
-if [ "${OSE_VERSION}" == "${OSE_MASTER}" ]; then
-  if [ "${BUILD_MODE}" == "online/stg" ] ; then
-    git checkout -q stage
-  fi
+if [[ "${OSE_VERSION}" == "${OSE_MASTER}" || "${OSE_VERSION}" == "${OSE_MASTER_BRANCHED}" ]]; then
   git remote add upstream git@github.com:openshift/origin.git --no-tags
   git fetch --all
 
+  if [[ "${BUILD_MODE}" == "online/stg" ]]; then
+    CURRENT_BRANCH="stage"
+    UPSTREAM_BRANCH="upstream/stage"
+  else if [[ "${OSE_VERSION}" == "${OSE_MASTER}" ]]; then
+    CURRENT_BRANCH="master"
+    UPSTREAM_BRANCH="upstream/master"
+  else if [[ "${OSE_VERSION}" == "${OSE_MASTER_BRANCHED}" ]]; then
+    CURRENT_BRANCH="enterprise-${OSE_VERSION}"
+    UPSTREAM_BRANCH="upstream/release-1.${MINOR}"
+  fi
```

Comment:
Now that 3.5 is released, that should be
`UPSTREAM_BRANCH="upstream/release-${MAJOR}.${MINOR}"`